lib/gds_api/publishing_api.rb

```diff
@@ -34,6 +36,31 @@ def get_content(content_id, params = {})
     get_json(content_url(content_id, params))
   end
 
+  # Return a live content item
```

Comment:
Probably worth expanding this comment to state that live means published or unpublished.